### DISCUSSION TITLE
GC105E found

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#13636] [Show and tell]
Just crossposting here, what I just posted on Facebook c:geo:

It seems the oldest not yet found geocache GC105E "Conch Shell Horn" was finally found using c:geo.
Congratulations on this FTF and thanks for using c:geo when going for it! 🙂
[https://s3.amazonaws.com/.../acd85f50-f322-4f10-91e6...](https://s3.amazonaws.com/gs-geo-images/acd85f50-f322-4f10-91e6-7011d0d64318.jpg?fbclid=IwAR1Ef9LdgEVwWMJ_mGwhEGNC9boPv5us-lJo0S0NR_Xw07Vhxaw4W9mdaoE)
[https://www.geocaching.com/seek/log.aspx?LID=1110437817...](https://www.geocaching.com/seek/log.aspx?LID=1110437817&IID=acd85f50-f322-4f10-91e6-7011d0d64318&fbclid=IwAR3FAE-8a9J-NYD0bI30PtAD7WBwTpnR4T_tOwMYy78nOhntEqhfZ0-_Ybo)
[https://www.geocaching.com/geocache/GC105E_conch-shell-horn](https://www.geocaching.com/geocache/GC105E_conch-shell-horn?fbclid=IwAR3Xq7GMrXVMr_9SWln3zh9m08sGmmBh3Dm5967tFdRGFvH-UbKqsfDtGi8)

**Comment by Narrat** [comment]
Interesting story.
And probably not surprising, but a whole lot of drama concerning/around this find...
